docs/color.md

```diff
@@ -0,0 +1,178 @@
+Found in `lua/pixelui/core/cl_color.lua`
+
+# PIXEL.DecToHex
+
+```lua
+function PIXEL.DecToHex(dec, zeros)
```

Comment:
If we're documenting the full library then it'd be worth moving this to a more appropriate section such as "Utilities". This is fine for now however.